sdk/keyvault/platform-matrix.json

```diff
@@ -0,0 +1,13 @@
+{
+  "matrix": {
+      "Groups": {
+          "ubuntu-18.04_HSM": {
+              "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general",
```

Comment:
Would prefer separate lines for properties (`Pool` on same line with `OSVmImage`).